bin/library.ps1

```diff
@@ -1964,16 +1964,22 @@ namespace Sqlcollective.Dbatools
 
                 string tempString = Name;
 
-                // Case: Default instance
+                // Case: Default instance | Instance by port
                 if (tempString.Split('\\').Length == 1)
                 {
-                    if (Regex.IsMatch(tempString, @":\d{1,5}$") && !Regex.IsMatch(tempString, Utility.RegexHelper.IPv6) && tempString.Split(':').Length == 2)
+                    if (Regex.IsMatch(tempString, @"[:,]\d{1,5}$") && !Regex.IsMatch(tempString, Utility.RegexHelper.IPv6) && ((tempString.Split(':').Length == 2) || (tempString.Split(',').Length == 2)))
```

Comment:
This...I like this! 
My first contribution to dbatools was trying to deal with this. 😃 
It's nice to see it done properly.